sitemedia/js/controllers/ittpanel_controller.js

```diff
@@ -28,5 +46,86 @@ export default class extends Controller {
                     break;
             }
         }
+
+        if (this.isDesktop()) {
+            if (this.transcriptionAndTranslationOpen()) {
+                // when transcription and translation are both opened, align their contents line-by-line
+                this.alignLines();
+            } else {
+                // when one of those two toggles is closed, remove inline styles from lines (alignment no longer needed)
+                this.transcriptionTarget
+                    .querySelectorAll("li")
+                    .forEach((li) => {
+                        li.removeAttribute("style");
+                    });
+                this.translationTarget.querySelectorAll("li").forEach((li) => {
+                    li.removeAttribute("style");
+                });
+            }
+        }
+    }
+
+    transcriptionAndTranslationOpen() {
+        // check toggle targets to find out whether both transcription and translated are open
+        return ["transcription-on", "translation-on"].every(
+            (id) =>
+                this.toggleTargets.find((target) => target.id === id)
+                    .checked === true
+        );
+    }
+
+    alignLines() {
+        // align each line of transcription to translation
+        const edLines = this.transcriptionTarget.querySelectorAll("li");
+        const trLines = this.translationTarget.querySelectorAll("li");
+        // only align as many lines as we need to
+        const [lessLines, moreLines] =
+            edLines.length < trLines.length
+                ? [trLines, edLines]
+                : [edLines, trLines];
```

Comment:
> Is it usually one set of content (translation?) that is longer, or is it possible for either set of content to be the longer one in any given line?

The latter is possible, I have seen it a few times. It will become much more common when Hebrew translations are entered.